### PR TITLE
updated document for MMA/AMA

### PR DESCRIPTION
**Pull request by HarshitArora8**

### CODE REVIEW

File: MMA Removal Utility/DeploymentFiles/MMARemovalUtilitySetup.ps1 [hunk 1]
```diff
@@ -405,7 +405,7 @@ function Set-AzTSMMARemovalUtilitySolutionRemediationIdentity {
             }
             
             # Grant User Identity Reader permission on target subscription(s).
-            Write-Host "Granting user-assigned identity 'Reader' and 'Virtual Machine Contributor' permission on target scope(s)..." -ForegroundColor $([Constants]::MessageType.Info)         
+            Write-Host "Granting user-assigned identity 'Reader', 'Virtual Machine Contributor' and Azure Arc ScVmm VM Contributor permission on target scope(s)..." -ForegroundColor $([Constants]::MessageType.Info)
```
[the remark this answers is not in the document]
Comment (TarunKrShukla) [reply MMA Removal Utility/DeploymentFiles/MMARemovalUtilitySetup.ps1:408]: and are we missing role for VMSS or  already covered in VM contributor role?

File: MMA Removal Utility/MultiTenantSetupInstallation.md [hunk 2]
```diff
@@ -289,8 +289,8 @@ Set-AzTSMMARemovalUtilityRunbook `
 This step involves creating SPN associated to identity for each tenant using Set-AzSKTenantSecuritySolutionMultiTenantIdentitySPN and helps in granting required role permissions to SPN on target scope(s) using Grant-AzSKAzureRoleToMultiTenantIdentitySPN.
 
 The AzTS MMA Removal Utility solution works in 2 phases:
-1. Discovers VMs and VM Extensions to identify the VMs from which MMA Agent will be removed for which setup requires ***Reader*** access on the scope being configured.
-2. Removes MMA Extensions from the eligible VMs for which setup requires ***Virtual Machine Contributor*** access on the scopes being configured. Scopes Configured can be a Tenant/ManagementGroup(s)/Subscription(s) or both ManagementGroup(s) and Subscription(s).
+1. Discovers VMs, VMSS and Azure Arc Servers and there Extensions to identify the services from which MMA Agent will be removed for which setup requires ***Reader*** access on the scope being configured.
```
Comment (TarunKrShukla) [inline MMA Removal Utility/MultiTenantSetupInstallation.md:292]: NIT: Discovers VMs, VMSS and Azure Arc Servers and there Extensions

Comment (TarunKrShukla) [reply MMA Removal Utility/MultiTenantSetupInstallation.md:292]: and repeated two times

File: MMA Removal Utility/README.md [hunk 3]
```diff
@@ -20,14 +20,14 @@ AzTS MMA Discovery and Removal Utility works in two phases:
 
 **1- Discovery Phase**
 
-In this phase, it will prepare the inventory of all Virtual Machines and their extensions to identify, which VMs have both MMA and AMA or only MMA agents installed. In next phase (i.e., 'Removal Phase'), users can choose to remove MMA extension from VMs having either both MMA and AMA agents or just MMA agent. 
+In this phase, it will prepare the inventory of all VM(s), VMSS(s), Azure Arc Servers and their extensions to identify, which services have both MMA and AMA or only MMA agents installed. In next phase (i.e., 'Removal Phase'), users can choose to remove MMA extension from the services having either both MMA and AMA agents or just MMA agent.
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:23]: services --> resources

Comment (TarunKrShukla) [reply MMA Removal Utility/README.md:23]: plz do this across all the page.

File: MMA Removal Utility/README.md [hunk 4]
```diff
@@ -192,8 +192,8 @@ The AzTS MMA Discovery and Removal utility is highly flexible and configurable.
 ### **2. What permissions does the AzTS MMA Discovery and Removal utility require?**
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:192]: Here also we need to add new permissions

File: MMA Removal Utility/README.md [hunk 5]
```diff
@@ -202,19 +202,20 @@ It is important to consider that if you choose to retain inventory or process lo
 ### **5. How much time AzTS MMA Discovery and Removal Utility will take to remove MMA agents?**
 Estimated average time required for a single run (both discovery and removal phase) of utility on a scope of approx. 100K VMs is around 30 Mins.
 
-### **6. How I can get list of all the VMs available in configured scope(s)?**
-VMs inventory is collected in Log Analytics workspace. To list all VMs discovered by utility, go to Log Analytics workspace created during setup --> Select Logs and run following query: 
+### **6. How I can get list of all the Azure services (VM,VMSS and Azure Arc servers) available in configured scope(s)?**
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:205]: How I can get list of all the Azure Virtual Machines, Virtual Machine scale sets and Azure Arc servers available in configured scope(s)

File: MMA Removal Utility/README.md [hunk 6]
```diff
@@ -202,19 +202,20 @@ It is important to consider that if you choose to retain inventory or process lo
...
+Services inventory is collected in Log Analytics workspace. To list all services discovered by utility, go to Log Analytics workspace created during setup --> Select Logs and run following query:
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:206]: Resource inventory (Virtual Machines, Virtual Machine scale sets and Azure Arc servers)

File: MMA Removal Utility/README.md [hunk 7]
```diff
@@ -202,19 +202,20 @@ It is important to consider that if you choose to retain inventory or process lo
...
 Inventory_CL
 | where TimeGenerated > ago(7d)
-| where ResourceType =~ "VirtualMachine"
 | summarize arg_max(TimeGenerated,*) by ResourceId = tolower(ResourceId)
 | extend OSType = tostring(parse_json(Metadata_s).OSType)
 ```
 
-### **7. How I can get list of all the VMs which have both MMA & AMA agent present and are eligible for removal phase?**
-VMs and Extensions inventory is collected in Log Analytics workspace. To list all the VMs which have both MMA & AMA agent present, go to Log Analytics workspace created during setup --> Select Logs and run following query: 
+### **7. How I can get list of all the VM(s),VMSS(s) and Azure Arc Servers which have both MMA & AMA agent present and are eligible for removal phase?**
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:215]: NIT: After , please use provide space.

File: MMA Removal Utility/README.md [hunk 8]
```diff
@@ -202,19 +202,20 @@ It is important to consider that if you choose to retain inventory or process lo
...
-VMs inventory is collected in Log Analytics workspace. To list all VMs discovered by utility, go to Log Analytics workspace created during setup --> Select Logs and run following query: 
+### **6. How I can get list of all the Azure services (VM,VMSS and Azure Arc servers) available in configured scope(s)?**
+Services inventory is collected in Log Analytics workspace. To list all services discovered by utility, go to Log Analytics workspace created during setup --> Select Logs and run following query: 
 
 ``` KQL
 Inventory_CL
 | where TimeGenerated > ago(7d)
-| where ResourceType =~ "VirtualMachine"
 | summarize arg_max(TimeGenerated,*) by ResourceId = tolower(ResourceId)
 | extend OSType = tostring(parse_json(Metadata_s).OSType)
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:212]: I hope OS type is being populated for all resource types properly.

File: MMA Removal Utility/README.md (hunk 7 above)
Comment (TarunKrShukla) [reply MMA Removal Utility/README.md:215]: This is at multiple places, please fix at all places,

File: MMA Removal Utility/README.md [hunk 9]
```diff
@@ -238,8 +239,82 @@ let virtualMachinesWithBothExtensions = virtualMachines
...
 
-### **8. How I can get list of all the VMs which MMA agent present?**
-VMs and Extensions inventory is collected in Log Analytics workspace. To list all the VMs which have both MMA agent present, go to Log Analytics workspace created during setup --> Select Logs and run following query: 
+For Virtual Machines Scale Set (VMSS):
+
+``` KQL
+let timeago = timespan(7d);
+let virtualMachines = Inventory_CL
+| where TimeGenerated > ago(timeago)
+| where ResourceId contains "microsoft.compute/virtualmachinescalesets"
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:248]: contains will also list extensions, we should use resource type with =~

Comment (TarunKrShukla) [reply MMA Removal Utility/README.md:248]: NIT: Variable name virtualMachines  should be changed appropriately in query.

File: MMA Removal Utility/README.md [hunk 10]
```diff
@@ -238,8 +239,82 @@ let virtualMachinesWithBothExtensions = virtualMachines
...
+| summarize Extensions = make_list(ExtensionType) by VMResourceID, OSType
+|where (Extensions contains "MicrosoftMonitoringAgent" or Extensions contains "OmsAgentForLinux") and (Extensions contains "AzureMonitorWindowsAgent" or Extensions contains "AzureMonitorLinuxAgent");
+virtualMachinesWithBothExtensions
+```
+
+For Azure Arc Servers:
+
+``` KQL
+let timeago = timespan(7d);
+let virtualMachines = Inventory_CL
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:270]: same as above query.

File: MMA Removal Utility/README.md [hunk 11]
```diff
@@ -238,8 +239,82 @@ let virtualMachinesWithBothExtensions = virtualMachines
...
+
+
+### **8. How I can get list of all the VMs,VMSS and Azure Arc Servers which have MMA agent present?**
+Azure resource data and Extensions inventory is collected in Log Analytics workspace. To list all the services which have both MMA agent present, go to Log Analytics workspace created during setup --> Select Logs and run following query: 
+
+For all services having MMA agent present:
+
+``` KQL
+let timeago = timespan(7d);
+let virtualMachines = Inventory_CL
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:298]: NIT: not just virtualMachines,

File: MMA Removal Utility/README.md [hunk 12]
```diff
@@ -238,8 +239,82 @@ let virtualMachinesWithBothExtensions = virtualMachines
...
+| where TimeGenerated > ago(timeago)
+| where ResourceType =~ "VMExtension" and Source_s =~ "AzTS_05_VMExtensionInventoryProcessor"
+| summarize arg_max(TimeGenerated,*) by ResourceId = tolower(ResourceId)
+| extend VMResourceID = tolower(substring(ResourceId,0,indexof(ResourceId, '/', 0, -1, 9 )))
+| extend ExtensionType = tostring(parse_json(Metadata_s).ExtensionType)
+| where ExtensionType =~ "OmsAgentForLinux" or ExtensionType =~ "MicrosoftMonitoringAgent"
+| project ResourceId, VMResourceID, ExtensionType, ExtensionResourceId = ResourceId;
+let virtualMachinesWithMMAExtensions = virtualMachines
+| join kind=inner  (virtualMachinesMMAExtensions) on VMResourceID
+| project VMResourceID, OSType, ExtensionResourceId, ExtensionType;
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:313]: Let's also project resource type

File: MMA Removal Utility/README.md [hunk 13]
```diff
@@ -262,6 +337,56 @@ let virtualMachinesWithMMAExtensions = virtualMachines
...
+virtualMachinesWithMMAExtensions
+```
+
+For Azure Arc Server(s) having MMA agent present:
+
+``` KQL
+let timeago = timespan(7d);
+let virtualMachines = Inventory_CL
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:369]: Same as above query, contains check and variable naming

File: MMA Removal Utility/README.md [hunk 14]
```diff
@@ -262,6 +337,56 @@ let virtualMachinesWithMMAExtensions = virtualMachines
...
+| extend ExtensionType = tostring(parse_json(Metadata_s).ExtensionType)
+| where ExtensionType =~ "OmsAgentForLinux" or ExtensionType =~ "MicrosoftMonitoringAgent"
+| project ResourceId, VMResourceID, ExtensionType, ExtensionResourceId = ResourceId;
+let virtualMachinesWithMMAExtensions = virtualMachines
+| join kind=inner  (virtualMachinesMMAExtensions) on VMResourceID
+| project VMResourceID, OSType, ExtensionResourceId, ExtensionType;
+virtualMachinesWithMMAExtensions
+```
+
+
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:389]: Any change needed FAQ#10

File: MMA Removal Utility/SingleTenantSetupInstallation.md [hunk 15]
```diff
@@ -60,8 +60,8 @@ Set-Prerequisites
 
 ### **Step 3 of 5. Setup remediation user-assigned managed identity**  
 The AzTS MMA Removal Utility solution works in 2 phases: 
-1. Discovers VMs and VM Extensions to identify the VMs from which MMA Agent will be removed for which setup requires ***Reader*** access on the scope being configured.
-2. Removes MMA Extensions from the eligible VMs for which setup requires ***Virtual Machine Contributor*** access on the scopes being configured. Scopes Configured can be a Tenant/ManagementGroup(s)/Subscription(s) or both ManagementGroup(s) and Subscription(s).
+1. Discovers VMs, VMSS and Azure Arc Servers and there Extensions to identify the services from which MMA Agent will be removed for which setup requires ***Reader*** access on the scope being configured.
```
Comment (TarunKrShukla) [inline MMA Removal Utility/SingleTenantSetupInstallation.md:63]: plz check if we need to add any new module or version as pre-requisite.

File: MMA Removal Utility/SingleTenantSetupInstallation.md [hunk 16]
```diff
@@ -60,8 +60,8 @@ Set-Prerequisites
...
+2. Removes MMA Extensions from the eligible Azure services for which setup requires ***Virtual Machine Contributor*** and ***Azure Arc ScVmm VM Contributor*** access on the scopes being configured. Scopes Configured can be a Tenant/ManagementGroup(s)/Subscription(s) or both ManagementGroup(s) and Subscription(s).
 
 For executing the above phases, setup requires a [User-Assigned Managed Identity (MI)](https: //docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) (identity part of the setup) which has 'Reader and Virtual Machine Contributor' access on target scopes configured.
```
Comment (TarunKrShukla) [inline MMA Removal Utility/SingleTenantSetupInstallation.md:66]: Here also, we need to add new required roles.

Comment (TarunKrShukla) [reply MMA Removal Utility/SingleTenantSetupInstallation.md:66]: Below in PowerShell command also in text we need to add new roles.

File: MMA Removal Utility/SingleTenantSetupInstallation.md [hunk 17]
```diff
@@ -239,8 +239,8 @@ Let's start!
 Consolidated installation can be done using Install-AzTSMMARemovalUtilitySolutionConsolidated.
 
 The AzTS MMA Removal Utility solution works in 2 phases: 
-1. Discovers VMs and VM Extensions to identify the VMs from which MMA Agent will be removed for which setup requires ***Reader*** access on the scope being configured.
-2. Removes MMA Extensions from the eligible VMs for which setup requires ***Virtual Machine Contributor*** access on the scopes being configured. Scopes Configured can be a Tenant/ManagementGroup(s)/Subscription(s) or both ManagementGroup(s) and Subscription(s).
+1. Discovers VMs, VMSS and Azure Arc Servers and there Extensions to identify the services from which MMA Agent will be removed for which setup requires ***Reader*** access on the scope being configured.
+2. Removes MMA Extensions from the eligible VMs, VMSS and Azure Arc Servers for which setup requires ***Virtual Machine Contributor*** and ***Azure Arc ScVmm VM Contributor*** access on the scopes being configured. Scopes Configured can be a Tenant/ManagementGroup(s)/Subscription(s) or both ManagementGroup(s) and Subscription(s).
```
Comment (TarunKrShukla) [inline MMA Removal Utility/SingleTenantSetupInstallation.md:243]: Here also there are some other places at which new permission in not updated.

File: MMA Removal Utility/DeploymentFiles/MMARemovalUtilitySetup.ps1 (hunk 1 above)
[the remark this answers is not in the document]
Comment (vaishnavipulluri) [reply MMA Removal Utility/DeploymentFiles/MMARemovalUtilitySetup.ps1:408]: NIT: Not all role names are surrounded by quotes, let maintain consistency in messages

File: MMA Removal Utility/MultiTenantSetupInstallation.md (hunk 2 above)
Comment (vaishnavipulluri) [reply MMA Removal Utility/MultiTenantSetupInstallation.md:292]: there -> their

File: MMA Removal Utility/DeploymentFiles/MMARemovalUtilitySetup.ps1
Comment (vaishnavipulluri) [inline MMA Removal Utility/DeploymentFiles/MMARemovalUtilitySetup.ps1:1]: if($assignmentError -eq $false)
            {
                Write-Host "Granted user-assigned identity 'Reader' and 'Virtual Machine Contributor' permission on target scope(s) successfully." -ForegroundColor $([Constants]::MessageType.Update)
            }

New roles should be included in such messages. Please check the entire flow once.

File: MMA Removal Utility/README.md [hunk 18]
```diff
@@ -192,8 +192,8 @@ The AzTS MMA Discovery and Removal utility is highly flexible and configurable.
 ### **2. What permissions does the AzTS MMA Discovery and Removal utility require?** 
 To operate effectively, the AzTS MMA Discovery and Removal utility creates a remediation identity. The specific permissions required for this identity depend on the scope and context. In the case of a single tenant, a user-assigned managed identity with the 'Reader' and 'Virtual Machine Contributor' roles is used. For multi-tenant scopes, an Azure AD App/SPN is employed. It is essential to grant the appropriate permissions to this identity at the configured scopes for successful execution.
 
-### **3. After MMA agents have been removed from all VMs, should I clean up the setup?**
-Once all MMA agents have been successfully removed from the VMs, it is recommended to perform a cleanup of the setup resources. It is important to note that while you may want to delete the setup resources, it is advisable to retain any inventory or process logs for future reference. The cleanup steps have been provided [here](./SetupCleanUp.md) in the documentation.
+### **3. After MMA agents have been removed from all VM(s),VMSS(s) and Azure Arc Servers should I clean up the setup?**
```
Comment (vaishnavipulluri) [inline MMA Removal Utility/README.md:195]: space missing after VM(s),

File: MMA Removal Utility/README.md (hunk 5 above)
Comment (vaishnavipulluri) [inline MMA Removal Utility/README.md:205]: NIT: How can I get .. ?

File: MMA Removal Utility/README.md (hunk 9 above)
Comment (vaishnavipulluri) [reply MMA Removal Utility/README.md:248]: Or can we just document the query once with generic names like inventory or resources in place of virtualMachines and ask user to replace with necessary configurations like resourceType = "VMSS" something like this if such changes are very less

File: MMA Removal Utility/SingleTenantSetupInstallation.md (hunk 15 above)
Comment (vaishnavipulluri) [inline MMA Removal Utility/SingleTenantSetupInstallation.md:63]: there -> their

File: MMA Removal Utility/README.md (hunk 9 above)
Comment (vaishnavipulluri) [inline MMA Removal Utility/README.md:248]: Lets also make sure we test all the queries once.

File: MMA Removal Utility/DeploymentFiles/MMARemovalUtilitySetup.ps1 (hunk 1 above)
[the remark this answers is not in the document]
Comment (HarshitArora8) [reply MMA Removal Utility/DeploymentFiles/MMARemovalUtilitySetup.ps1]: 1. VMSS is already covered in VM contributor role, PFA screenshot for reference:
image
2. Added quotes as well.

File: MMA Removal Utility/MultiTenantSetupInstallation.md (hunk 2 above)
Comment (HarshitArora8) [reply MMA Removal Utility/MultiTenantSetupInstallation.md]: Fixed

File: MMA Removal Utility/README.md (hunk 4 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md:192]: Added

File: MMA Removal Utility/README.md (hunk 5 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md]: Added

File: MMA Removal Utility/README.md (hunk 6 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md]: Added

File: MMA Removal Utility/README.md (hunk 7 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md]: Fixed and verified in all the files.

File: MMA Removal Utility/README.md (hunk 14 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md:389]: Fixed the FAQ#10

File: MMA Removal Utility/SingleTenantSetupInstallation.md (hunk 15 above)
Comment (HarshitArora8) [reply MMA Removal Utility/SingleTenantSetupInstallation.md]: Updated and Verified.

File: MMA Removal Utility/README.md (hunk 3 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md]: Fixed in all the pages.

File: MMA Removal Utility/SingleTenantSetupInstallation.md (hunk 15 above)
Comment (HarshitArora8) [reply MMA Removal Utility/SingleTenantSetupInstallation.md]: No module addition required.

File: MMA Removal Utility/SingleTenantSetupInstallation.md (hunk 16 above)
Comment (HarshitArora8) [reply MMA Removal Utility/SingleTenantSetupInstallation.md:66]: Added in all the required places.

File: MMA Removal Utility/DeploymentFiles/MMARemovalUtilitySetup.ps1
Comment (HarshitArora8) [reply MMA Removal Utility/DeploymentFiles/MMARemovalUtilitySetup.ps1:1]: Added new roles in all the places required.

File: MMA Removal Utility/README.md (hunk 18 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md]: fixed and verified.

File: MMA Removal Utility/README.md (hunk 5 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md]: Fixed in all the places

File: MMA Removal Utility/README.md (hunk 8 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md:212]: Yes.

File: MMA Removal Utility/SingleTenantSetupInstallation.md (hunk 17 above)
Comment (HarshitArora8) [reply MMA Removal Utility/SingleTenantSetupInstallation.md:243]: updated.

File: MMA Removal Utility/README.md (hunk 9 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md]: All queries are tested

File: MMA Removal Utility/README.md (hunk 11 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md:298]: Added

File: MMA Removal Utility/README.md (hunk 13 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md:368]: Done

File: MMA Removal Utility/README.md (hunk 10 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md:270]: Added

File: MMA Removal Utility/README.md (hunk 12 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md]: Added

File: MMA Removal Utility/README.md (hunk 14 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md:388]: Fixed variable naming in LA queries.

**Review by vaishnavipulluri — APPROVED** [review]

File: MMA Removal Utility/README.md [hunk 19]
```diff
@@ -105,7 +105,7 @@ Update-AzTSMMARemovalUtilityDiscoveryTrigger `
 
 ```
 ### Schedule or Trigger Removal phase
-Removal phase is disabled by default. Users need to enable/schedule it after validating the inventory of VMs list prepared in the Discovery phase. Command **Update-AzTSMMARemovalUtilityRemovalTrigger** can be used to:
+Removal phase is disabled by default. Users need to enable/schedule it after validating the inventory of resource list prepared in the Discovery phase. Command **Update-AzTSMMARemovalUtilityRemovalTrigger** can be used to:
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:108]: NIT: inventory of resource list --> inventory of resources

File: MMA Removal Utility/README.md [hunk 20]
```diff
@@ -262,6 +336,56 @@ let virtualMachinesWithMMAExtensions = virtualMachines
...
 virtualMachinesWithMMAExtensions
 ```
+
+For VMSS(s) having MMA agent present:
+
+``` KQL
+let timeago = timespan(7d);
+let vmss = Inventory_CL
+| where TimeGenerated > ago(timeago)
+| where ResourceType contains "VirtualMachineScaleSets"
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:346]: As suggested earlier avoid contains check, if =~ operator can be utilized.

File: MMA Removal Utility/README.md [hunk 21]
```diff
@@ -262,6 +336,56 @@ let virtualMachinesWithMMAExtensions = virtualMachines
...
+vmssWithMMAExtensions
+```
+
+For Azure Arc Server(s) having MMA agent present:
+
+``` KQL
+let timeago = timespan(7d);
+let hybridVirtualMachines = Inventory_CL
+| where TimeGenerated > ago(timeago)
+| where ResourceType contains "HybridCompute"
```
Comment (TarunKrShukla) [inline MMA Removal Utility/README.md:370]: same

File: MMA Removal Utility/README.md (hunk 19 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md]: Fixed

File: MMA Removal Utility/README.md (hunk 20 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md]: Added =~ in all the places and tested the query in LA

File: MMA Removal Utility/README.md (hunk 21 above)
Comment (HarshitArora8) [reply MMA Removal Utility/README.md]: Added =~ in all the places and tested the query in LA

**Review by TarunKrShukla — APPROVED** [review]

**Pull request merged**